openmrs/src/test/java/org/motechproject/openmrs/validation/ConfigValidatorTest.java

```diff
@@ -20,6 +21,7 @@
     public void shouldPassTheValidationIfAllTheFieldsAreNonEmpty() {
         Config config = new Config();
```

Comment:
there is a lot of duplicated code here, please use constructor or a private method for setting all these fields